### PR TITLE
Fix manage creditcards for SW 5.7 

### LINKED ISSUE #118: The "Manage Credit Cards" button in the frontend does not work
**How does the system behave currently?**

When logging in as a customer in the frontend and selecting the Manage Credit Card button, the following error message appears.

image

`Fatal error: Uncaught ArgumentCountError: Too few arguments to function Shopware_Controllers_Frontend_Account::__construct(), 0 passed in /var/www/engine/Library/Enlight/Controller/Dispatcher/Default.php on line 441 and exactly 2 expected in /var/www/engine/Shopware/Controllers/Frontend/Account.php:74`

Therefore, it is currently not possible for the customer to manage his credit cards. 

**How should the system behave?**

The correct interface should be displayed.

### PR DESCRIPTION
**Pull request by JanRogge**
With 5.7 the construct of `Shopware_Controllers_Frontend_Account` was changed to now need two parameters.

This PR retrieves the two parameters and passes them to the parent constructor.

Fixes #118

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -38,5 +38,11 @@
         "exclude-from-classmap": [
             "Polyfill"
         ]
+    },
+    "config": {
+        "allow-plugins": {
+            "composer/installers": true,
+            "viison/composer-git-hooks-installer-plugin": true
+        }
```
Comment (stefanheppenheimer) [inline composer.json:46]: This change should be a separate PR.

File: Controllers/Frontend/StripePaymentAccount.php [hunk 2]
```diff
@@ -13,6 +13,16 @@
...
 {
+    /**
+     * @inheritdoc
+     */
+    public function __construct()
+    {
+        $config = Shopware()->Container()->get('config');
+        $captchaValidator = Shopware()->Container()->get('shopware.captcha.validator');
+        parent::__construct($config, $captchaValidator);
+    }
```
Comment (stefanheppenheimer) [inline Controllers/Frontend/StripePaymentAccount.php:24]: Is this change backwards compatible?

Comment (JanRogge) [reply Controllers/Frontend/StripePaymentAccount.php:24]: I've tested this with Shopware 5.2.7 and it seems to work just fine.

**Review by stefanheppenheimer — APPROVED** [review]
